delete.go

```diff
@@ -188,6 +197,11 @@ func (b *DeleteBuilder) What(what ...string) *DeleteBuilder {
 	return b
 }
 
+func (b *DeleteBuilder) Using(table string) *DeleteBuilder {
```

Comment:
Could you please add a comment here with explicit explanation that this method is PostgreSQL-specific?